source/android-source/pilgrimunitysdk/src/main/java/com/foursquare/pilgrimunitysdk/Utils.java

```diff
@@ -147,42 +147,29 @@ private static JSONObject visitJson(@NonNull Visit visit) throws JSONException {
             json.put("_venue", venueJson(visit.getVenue()));
         }
 
-        if (visit.getOtherPossibleVenues() != null) {
-            JSONArray otherPossibleVenuesArray = new JSONArray();
-            for (Venue venue : visit.getOtherPossibleVenues()) {
-                otherPossibleVenuesArray.put(venueJson(venue));
-            }
-            json.put("_otherPossibleVenues", otherPossibleVenuesArray);
+        JSONArray otherPossibleVenuesArray = new JSONArray();
+        for (Venue venue : visit.getOtherPossibleVenues()) {
```

Comment:
did we change this to NonNull?

Reply:
Yeah it's non-null in 2.2.0 for android, however it's nullable for iOS, kinda weird